src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json

```diff
@@ -34,5 +34,12 @@
             "priority":9996
         },
         "OP":"SET"
+    },
+    {
+        "ACL_RULE_TABLE:dataacl:default_rule":{
+            "PACKET_ACTION":"DROP",
```

Comment:
Didn't quite get it. Shouldn't we have an explicit ACL rule to allow lldp/arp packets?